docs/references/api/resource_embedding.rst

```diff
@@ -318,10 +321,231 @@ Computed relationships have good performance as their intended design enable `in
 
   - Make sure to correctly label the ``to-one`` part of the relationship. When using the ``ROWS 1`` estimation, PostgREST will expect a single row to be returned. If that is not the case, it will unnest the embedding and return repeated values for the top level resource.
 
+.. _complex_rels:
+
+Complex Relationships
+=====================
+
+In most cases, you need to use :ref:`computed_relationships` to embed a resource with itself.
```

Comment:
Maybe this should go or change to something like:

```suggestion
In most cases, you need to use :ref:`computed_relationships` to embed complex relationships.
```

It was referring to the Recursive Relationships before.